Assets/Enemies/EnemyHitSystem.cs

```diff
@@ -11,13 +11,13 @@ void Start()
 		hitPool.Initialize();
 	}
 
-	public void OnEnemyHit(EnemyTypeDefinition enemyDef, Vector2 pos)
+	public void OnEnemyHit(EnemyTypeDefinition enemyDef, Vector2 pos, AudioClip clip)
```

Comment:
Instead of adding an AudioClip parameter should we just use the passed in enemyDef to play the sound?